src/main/java/cloud/fogbow/ras/core/plugins/interoperability/azure/network/sdk/AzureVirtualNetworkOperationSDK.java

```diff
@@ -104,15 +113,15 @@ private void subscribeVirtualNetworkCreation(Observable<Indexable> virtualNetwor
                 .subscribe();
     }
 
-    public AzureGetVirtualNetworkRef doGetInstance(String resourceName, AzureUser azureUser)
+    public AzureGetVirtualNetworkRef doGetInstance(String resourceName, String orderName, AzureUser azureUser)
```

Comment:
It is not necessary to pass the orderName by parameter. The resource name is contained in the resource tag.